src/server/connection.rs

```diff
@@ -3139,10 +3151,7 @@ async fn start_ipc(
                 #[cfg(target_os = "windows")]
                 {
                     log::debug!("Start cm");
-                    res = crate::platform::run_as_user(
-                        args.clone(),
-                        Some(crate::platform::get_current_process_session_id()),
-                    );
+                    res = crate::platform::run_as_user(args.clone(), user_session_id);
```

Comment:
https://github.com/rustdesk/rustdesk/pull/6825#discussion_r1476928541 Initially it was to solve this problem.

Reply:
https://github.com/rustdesk/rustdesk/pull/6825#discussion_r1479359255

This took care of it:
```rust
let mut session_id = get_current_process_session_id();
```